cypress/integration/main_spec.js

```diff
@@ -0,0 +1,81 @@
+import { O_TRUNC } from "constants";
+import { getFriendlyExpiryDescriptionForHistory } from "../../src/lib/utils";
+
+describe('Probe Dictionary Main Component', function() {
+  it('successfully loads', function() {
+    cy.visit('/');
```

Comment:
I don't know a lot about Cypress, but does this check that the React app renders? If not, that might be another good test to add. I've had issues where `*index.html* will be served up as expected, but the app won't render due to some issue.

Reply:
It doesn't have to be super complex. I often just check that the top-level React element is present.

Reply:
Great call!